styles.css

```diff
@@ -0,0 +1,84 @@
+* {
+    font-family: 'Poppins', sans-serif;
+}
+
+.main {
+    background-image: url("assets/header-shapes-mobile.png");
+    background-repeat: no-repeat;
+    width: 375px;
+}
+
+.nav-bar {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    background: #fff;
+    height: 48px;
+    width: 100%;
+}
+
+.nav-list {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    padding: 5px;
+    font-size: 15px;
+}
+
+.nav-items {
+    list-style: none;
+    margin: 15px 0 0 20px;
+}
+
+.logo {
+    position: static;
+    width: 76px;
+    height: 30px;
+    margin: 28px 0 0 24px;
+}
+
+.container {
+    position: static;
+    width: 327px;
+    padding: 0 24px;
+}
+
+h1 {
+    color: #172B4D;
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    text-align: left;
+    margin: 12px 0px;
+    width: 327px;
+}
+
+p {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    color: #344563;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    width: 327px;
+}
+
+.foot {
+    color: #7F8CFF;
+    width: 117px;
+    height: 12px;
+    padding: 12px 0 0 24px;
+}
+
```

Comment:
- [ ] Kindly fix the design according to Figma. Make sure to follow the font size, spacing, font family, margins according to the template you have chosen.
- Your design:
<img width="356" alt="Screenshot 2022-03-16 at 11 30 53 PM" src="https://user-images.githubusercontent.com/70269507/158690351-3f5df5f7-8b8c-4564-ae65-d52b46fef5e6.png">

- Figma design:
<img width="246" alt="Screenshot 2022-03-16 at 11 32 15 PM" src="https://user-images.githubusercontent.com/70269507/158690374-c36a8ffd-16da-444c-85ac-08cb613790fe.png">